### PR TITLE
bump version to 2.5.0 and update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
LGTM

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,7 +3,37 @@ For more detailed information, please see the git log.
...
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyblocks provides 154 software-specific easyblocks and 28 generic easyblocks.
+The latest version of easybuild-easyblocks provides 157 software-specific easyblocks and 28 generic easyblocks.
+
+v2.5.0 (December 17th 2015)
+---------------------------
+
+feature + bugfix release
+- update easyblocks for Intel tools to support 2016 versions (#691, #745, #756, #777)
```
Comment (wpoely86) [inline RELEASE_NOTES]: add something about chosing components?

**Comment by wpoely86** [comment]
:+1:

File: RELEASE_NOTES (hunk 1 above)
Comment (boegel) [inline RELEASE_NOTES]: done

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**